tech/languages/python/multiple-pythons.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Multiple Pythons
+subsection: python
+order: 8
+---
+
+# Multiple Python Interpreters
+
+If you are working on a piece of Python software, you probably want to test it
+on multiple Python interpreters. On Fedora, that's easy, all you have to do is
+use `dnf` to install whta you need. Currently Fedora has the following Pythons
+ready for you in the repositories:
+
+ * CPython 3.6
+ * CPython 3.5
+ * CPython 3.4
+ * CPython 3.3
+ * CPython 2.7
+ * CPython 2.6
+ * PyPy
+ * PyPy 3
+ * Jython
+ * MicroPython
+
+Quite a nest, isn't it?
+You can install them like this:
+
+```bash
+sudo dnf install python33  # to get CPython 3.3
+sudo dnf install python34  # to get CPython 3.4
+sudo dnf install python26  # to get CPython 2.6
+sudo dnf install pypy pypy3 jython python35  # to get more at once
+```
+
+After that, you can run an interactive console or your script with let's say
+CPython 3.4:
+
+```bash
+python3.4
+```
+
+Or get pip and use it to install packages:
+
+```bash
+python3.4 -m ensurepip --user
+python3.4 -m pip install --user <package>
+```
+
+Or even create virtualenvs:
+
+```bash
+python3.4 -m venv <dir>
+```
+
+Note that most of the CPython versions, other than those provided by the
+`python2` and `python3` packages, are only provided for developer's convenience
+so you can run your test suites. Critical security bugs might not be always
+fixed and those interprets are not intended for production.
+
+## Getting it and running it all with tox
```

Comment:
This is the main use case. I'd put it after the list of available interpreters, but before mentioning how to install things manually.
IMO the section needs a short introduction on what tox is.